makefiles/blob.inc.mk

```diff
@@ -0,0 +1,50 @@
+#
+# makes any file in BLOBS available via '#include "blob/<filename>.h"
+#
+# Usage:
+#
+# Add this to an application or module Makefile:
+#
+#     BLOBS += foo.ext
+#
+# Then include in C file or header:
+#
+#    #include "blob/foo.ext.h"
+#
```

Comment:
Could you also include the generated variable names in the example here?
Something like:
```
#include "blob/foo.ext.h"

print((char *)foo_ext, foo_ext_len);
```

Reply:
I've added some docs.